src/routes/start/umgebung/+page.svelte

```diff
@@ -7,11 +7,18 @@
     stopLocationWatch,
   } from "@stores/location";
 
-  import { niceDistance } from "@/utils/geoCoords";
+  import { niceDistance, distance } from "@/utils/geoCoords";
 
   import { data as arExperienceData } from "@data/arExperiences";
   import { onMount } from "svelte";
 
+  $: sortedArExperienceData = arExperienceData.sort((a, b) => {
+    const distanceA = distance(a.geoLocation, $currentLocation);
+    const distanceB = distance(b.geoLocation, $currentLocation);
+
+    return distanceA - distanceB;
+  });
```

Comment:
Vielleicht könnte man die Sortierfunktion etwas vereinfachen, wie folgt:
``
  $: sortedArExperienceData = arExperienceData.sort((a, b) => {
    return distance(a.geoLocation, $currentLocation) - distance(b.geoLocation, $currentLocation);
  });
``

Reply:
Damit verliert der Block jedoch an Lesbarkeit, da zu viele Operationen in einer Zeile/Aufruf passieren. Daher diese Wahl